src/Filterable.ts

```diff
@@ -123,25 +123,25 @@ export interface Filterable3C<F extends URIS3, U, L> extends Functor3C<F, U, L>,
  * @since 1.7.0
  */
 export function getPartitionMapFromSeparate<F extends URIS3, U, L>(
-  F: Functor3C<F, U, L> & Pick<Compactable3C<F, U, L>, 'separate'>
+  F: Pick<Functor3C<F, U, L> & Compactable3C<F, U, L>, 'URI' | 'map' | '_U' | '_L' | 'separate'>
```

Comment:
mmh this is becoming a bit too messy don't you think?

Reply:
Well, default implementations are likely to be used only by developers building custom ADTs who are already confident with a tricky HKTs in fp-ts. So I don't think there will be any problems to understand this signature because they should already know 'URI', '_U', '_L' and others :) In ther worst case we can add `@example` tag to each default function.